core-bundle/src/Cache/ContaoCacheWarmer.php

```diff
@@ -199,6 +201,14 @@ private function generateTemplateMapper(string $cacheDir): void
         );
     }
 
+    private function generateColumnCastTypes(string $cacheDir): void
+    {
+        $this->filesystem->dumpFile(
+            Path::join($cacheDir, 'contao/config/column_cast_types.php'),
```

Comment:
Just a nitpick, I would like `column-types.php` better. 🙈

Reply:
Renamed in 336ec3db7663371f3c6ad74aa6c67d32bd280d5c